pkg/volume/csi/csi_client.go

```diff
@@ -162,8 +161,8 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 			return true, nil
 		}
 		// kubelet plugin registration service not implemented is a terminal error, no need to retry
```

Comment:
this comment should be removed

Reply:
@msau42 
Your comment has been addressed.